README

```diff
@@ -0,0 +1,32 @@
+python-tx-tftp
```

Comment:
Is it really neccessary to add this file to repository? IMO, since README.markdown is the "original", it has to stay and README (which is derived from README.markdown) has to be added to .gitignore